lua/telescope/_extensions/ui-select.lua

```diff
@@ -9,15 +9,83 @@ return require("telescope").register_extension {
     local conf = require("telescope.config").values
     local actions = require "telescope.actions"
     local action_state = require "telescope.actions.state"
+    local strings = require "plenary.strings"
+    local entry_display = require "telescope.pickers.entry_display"
+
+    local specific_opts = {
+      ["Code actions:"] = {
+        make_indexed = function(items)
+          local indexed_items = {}
+          local widths = {
+            idx = 0,
+            command_title = 0,
+            client_name = 0,
+          }
+          for idx, item in ipairs(items) do
+            local client = vim.lsp.get_client_by_id(item[1])
```

Comment:
we could  stash this. But this could also be stashed on neovim side already. Need to check source code